code/game/objects/items/devices/radio/headset.dm

```diff
@@ -290,6 +290,22 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 /obj/item/radio/headset/silicon/can_receive(freq, level)
 	return ..(freq, level, TRUE)
 
+/obj/item/radio/headset/clown
+	name = "Clowning Headset"
+	desc = "A headset with a clowning filter, honk!"
+	icon_state = "honk_headset"
+	honken = TRUE  // honk filter
+	use_honken = TRUE
+
+/obj/item/radio/headset/clown/talk_into_impl(atom/movable/M, message, channel, list/spans, datum/language/language, list/message_mods)
+	if(honken)
+		spans |= SPAN_CLOWN
+	. = ..()
+
+/obj/item/radio/headset/clown/examine(mob/user)
+	. = ..()
+	. += "The honkening filter is currently [honken ? "On" : "Off"], use Alt-Click to toggle."
```

Comment:
gah wait, fix this

Reply:
probably just remove

Reply:
wait why? the command loudmode has a similar examine str

Reply:
`use Alt-Click to toggle.` this is not accurate, is it?

Reply:
oh oops